electron-app/magnifier/rust-sampler-manager.test.ts

```
@@ -0,0 +1,519 @@
+    if (manager.isRunning()) {
+      await manager.stop();
+    }
+    vi.clearAllMocks();
+  });
+
+  describe('start', () => {
+    it('should spawn the rust sampler process', async () => {
+      const onData = vi.fn();
+      const onError = vi.fn();
+
+      const startPromise = manager.start(9, 20, onData, onError);
```

ESLint

[error] 'startPromise' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)